data/postgres/schema.sql

```diff
@@ -0,0 +1,17 @@
+DROP DATABASE earnings;
+CREATE DATABASE earnings;
+\c earnings
+
+DROP TABLE IF EXISTS tickers;
+
+CREATE TABLE tickers (
+  id SERIAL PRIMARY KEY,
+  ticker  VARCHAR(50) UNIQUE NOT NULL,
+  name VARCHAR(50) NOT NULL,
+  earnings TEXT []
```

Comment:
why not have a separate table for earnings?

Reply:
I didn't have a specific reason other than trying to keep things super simple